sympy/vector/coordsysrect.py

```diff
@@ -704,3 +896,30 @@ def _check_strings(arg_name, arg):
                 raise TypeError(errorstr)
     except:
         raise TypeError(errorstr)
+
+
+def get_metric(relation_lambda):
+    from sympy.abc import a, x, y, z
+
+    variables = x, y, z
+    if len(relation_lambda.args[0]) == 3:
+        relations = relation_lambda(x, y, z)
+    elif len(relation_lambda.args[0]) == 4:
+        relations = relation_lambda(a, x, y, z)
+
+    jacobian = Matrix([[relation.diff(var) for var in variables]
+                       for relation in relations])
+
+    return simplify(jacobian.T * eye(3) * jacobian)
+
+
+def get_lame_lambda(relation_lambda):
```

Comment:
Haha.  I thought about that.  It represents Lamé, the guy the coefficients are named after.